docs/outlook/add-in-requirements.md

```diff
@@ -17,11 +17,11 @@ For Outlook add-ins to load and function properly, there are a number of require
    - Outlook 2016 or later on Mac
    - Outlook on iOS
    - Outlook on Android
-   - Outlook on the web for Exchange 2016 or later and Office 365
+   - Outlook on the web for Exchange 2016 or later
    - Outlook on the web for Exchange 2013
    - Outlook.com
 
-- The client must be connected to an Exchange server or Microsoft 365 using a direct connection. When configuring the client, the user must choose an **Exchange**, **Office 365**, or **Outlook.com** account type. If the client is configured to connect with POP3 or IMAP, add-ins will not load.
+- The client must be connected to an Exchange server or Microsoft 365 using a direct connection. When configuring the client, the user must choose an **Exchange**, **Office**, or **Outlook.com** account type. If the client is configured to connect with POP3 or IMAP, add-ins will not load.
```

Comment:
Is there anyway to confirm what the current value is? Was it changed from "Office 365"?